services/blockchain-indexer/shared/indexer/genesisBlock.js

```diff
@@ -133,9 +137,53 @@ const indexPosModuleAssets = async (dbTrx) => {
 	await indexPosStakesInfo(numStakers, dbTrx);
 };
 
+const indexNFTSubstore = async (nftSubstoreLength, dbTrx) => {
+	const nftTable = await getNFTTable();
+
+	if (nftSubstoreLength > 0) {
+		const nftSubstore = (await requestAll(
+			requestConnector,
+			'getGenesisAssetByModule',
+			{ module: MODULE.NFT, subStore: MODULE_SUB_STORE.NFT.NFT_SUB_STORE },
+			nftSubstoreLength,
+		)).NFTSubstore;
+
+		const nftEntries = nftSubstore.map(nftInfo => ({
+			nftID: nftInfo.nftID,
+			owner: nftInfo.owner,
```

Comment:
Should we also index NFT attributes here?

Reply:
Doesn't seem necessary.